src/components/side_panel/conditional_formatting/cf_preview_list/cf_preview_list.ts

```diff
@@ -58,8 +54,36 @@ export class ConditionalFormatPreviewList extends Component<Props, SpreadsheetCh
     });
   }
 
+  onAddConditionalFormat() {
+    const sheetId = this.env.model.getters.getActiveSheetId();
+    const zones = this.env.model.getters.getSelectedZones();
+    const cf: Omit<ConditionalFormat, "ranges"> = {
```

Comment:
nitpick, if you have a cool idea for a name, an alias could be nice since we use it at 3 different places already